react-hot-button-component/src/hot-button.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+let clickCount = 0;
```

Comment:
This should be a property of state, otherwise you can't mount more than one independent button.